.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. When I launch the docker file
+2. And then I run this command
+3. I get the following weird thing
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment:**
+ - OS: [e.g. ubuntu 20.04]
+ - Python version: [e.g. 3.9 ]
+ - Cuda version: [e.g. 11.6 ]
+ - Pytorch version: [e.g. 1.13.1]
+ If you are running things in the provided docker file just say that.
```

Comment:
Docker files don't 100% guarantee replicability -- because they install things from APT etc may get different versions depending on when it's built. Suggest uploading pre-built images to Docker hub and having people reference tags, or if no pre-built images have them include versions in issue.